transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SQLiteEventStore.java

```diff
@@ -587,6 +587,9 @@ public void recordLogEventDropped(
       long eventsDroppedCount, LogEventDropped.Reason reason, String logSource) {
     inTransaction(
         db -> {
+          if (isStorageAtGlobalLimit()) {
```

Comment:
It looks like the `isStorageAtLimit` method will act differently now, i.e. it will account health metrics, effectively reducing the size to 8-10MB(depending on how many health metric events we have), are we ok with that?